src/pudl_archiver/orchestrator.py

```diff
@@ -239,7 +244,9 @@ async def run(self) -> RunSummary | Unchanged:
             for name in files_to_delete
         ]
 
-        self.new_deposition = await self.depositor.get_record(self.new_deposition.id_)
+        self.new_deposition = await self.depositor.get_draft_record(
```

Comment:
I'm pleased that the changes to `orchestrator.py` are so light / the interface to the depositor didn't change much!

Once Zenodo stabilizes and the docs exist, we can maybe think about what the depositor interface should look like going forward... something along the lines of:

* create new deposition
* update the deposition file contents (all the draft state management might want to be hidden from the outside)
* publish the deposition

But - since we're most likely going to be tied to Zenodo for the forseeable future, and I don't expect them to be changing the API willy-nilly anytime soon, maybe that's not a super useful refactor.